product_packaging_dimension/migrations/16.0.1.1.0/pre-migrate.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+def _move_max_weight(env):
+    """
+    Move max_weight value to weight if not defined
+    """
+    query = """
+        UPDATE product_packaging
+            SET weight = max_weight
+            WHERE weight IS NULL OR weight = 0.0
+            AND max_weight IS NOT NULL
```

Comment:
Maybe only for the product packaging linked to a product.
```suggestion
            AND max_weight IS NOT NULL
            AND product_id IS NOT NULL
```

Reply:
Done